include/lo2s/trace/trace.hpp

```diff
@@ -93,11 +95,12 @@ class Trace
 
     void process_update_executable(pid_t pid, const std::string& exe_name);
 
-    otf2::writer::local& sample_writer(pid_t pid, pid_t tid);
-    otf2::writer::local& cpu_writer(int cpuid);
-    otf2::writer::local& metric_writer(pid_t pid, pid_t tid);
-    otf2::writer::local& metric_writer(const std::string& name);
+    otf2::writer::local& thread_sample_writer(pid_t pid, pid_t tid);
+    otf2::writer::local& cpu_sample_writer(int cpuid);
+    otf2::writer::local& thread_metric_writer(pid_t pid, pid_t tid);
+    otf2::writer::local& named_metric_writer(const std::string& name);
```

Comment:
Please verify that you caught all instances, where you need to change that name. I think there is one missing for metric/x86_energy.